keyboards/ziz0gaming19/c13x/keymaps/default/keymap.c

```diff
@@ -0,0 +1,89 @@
+#include QMK_KEYBOARD_H
+
+// Layers
+enum layers {
+    _BASE,
+    _FN,
+    _LAYER2,
+    _LAYER3
```

Comment:
based on your keymap, `_FN` should be the highest layer
```suggestion
    _BASE,
    _LAYER2,
    _LAYER3,
    _FN
```